lib/suspenders/app_builder.rb

```diff
@@ -432,16 +432,25 @@ def gitignore_files
 
     def init_git
       run 'git init'
+      template '.overcommit.yml', '.overcommit.yml'
+      bundle_command 'exec overcommit --install'
+      bundle_command 'exec overcommit --sign'
+      bundle_command 'exec overcommit --sign pre-commit'
```

Comment:
With overcommit, does every developer who clones the repo need to run anything in order to get it to bootstrap the  hooks in their local repo? Or does it magically do that somehow? (If so: any idea how?)

Reply:
Nope! Now that the `overcommit` gem is added and these steps run, all the user needs to do is commit normally and they get all the features of the pre-commit hooks. These steps are what installs the hooks in the `.git` folder